pkg/graph/schema.resolvers.go

```diff
@@ -462,6 +462,19 @@ func (r *mutationResolver) CreateAccessibilityRequestDocument(ctx context.Contex
 	}, nil
 }
 
+func (r *mutationResolver) CreateAccessibilityRequestNote(ctx context.Context, input model.CreateAccessibilityRequestNoteInput) (*model.CreateAccessibilityRequestNotePayload, error) {
```

Comment:
Could you add a corresponding test for this in `schema.resolvers_accessibility_request_test`?